userspace/falco/event_drops.h

```diff
@@ -29,7 +29,7 @@ limitations under the License.
 // detecting a syscall event drop.
 enum class syscall_evt_drop_action : uint8_t
 {
-	IGNORE = 0,
+	DISREGARD = 0,
```

Comment:
It's not really clear to me why we need this. Is it because just having anything that is called `IGNORE` (even if it's a scoped enum) ends up being a compilation error? Seems a little weird but not impossible!

Reply:
The problem is this define: `winbase.h:479:#define IGNORE 0` :/